src/mediaelement.js

```diff
@@ -59,7 +59,7 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
     },
 
     isPaused: function () {
-        return this.media.paused;
+        return this.media && this.media.paused;
```

Comment:
@katspaugh It was called internally. How do you think if we will create [Null Object](https://sourcemaking.com/design_patterns/null_object) and set it to media when media it is destroued. Player will work like a cassete player when you turn it on without cassete.